tests/api-watch-diff/test.ml

```diff
@@ -2,3 +2,196 @@ let%expect_test "test_diff_interface" =
   let result = Api_watch_diff.diff_interface ~reference:[] ~current:[] in
   Format.printf "%b" result;
   [%expect {|false|}]
+
+open Types
+
+(* Signature for type t:
+     > type t = int *)
+
+let t_sig =
+  Sig_type
+    ( Ident.create_persistent "t",
+      {
+        type_params = [];
+        type_arity = 0;
+        type_kind = Type_abstract;
+        type_private = Public;
+        type_manifest =
+          Some
+            (Transient_expr.type_expr
+               (Transient_expr.create (Tvar (Some "int")) ~level:0 ~scope:0
```

Comment:
I think you are hitting the same problem I explained here https://github.com/NathanReb/ocaml-api-watch/pull/36#discussion_r1529470985 #37 

The gist is that these should be type constructors (`Tconst`) and not type variables (`Tvar`) :)

Reply:
@shonfeder Thanks for this approach! I think the it worked. Also, we would have to use ```Ident.create_predef``` instead of ```Ident.create_persistent``` when giving the path to ```Tconstr``` because the ```Global``` identifier doesn't let the test detect the changes made to the ```value```.